src/localization/locales/en.json

```diff
@@ -625,7 +626,45 @@
             "on": "Yes",
             "off": "No"
         },
-        "error": "Failed to monitor events. {{error}}"
+        "error": "Failed to monitor events. {{error}}",
+        "simulation": {
+            "header": "Simulate a device using CLI in Azure Cloud Shell",
+            "cloudShell": {
+                "imageDescription": "Launch Azure Cloud Shell",
+                "textDescription": "Click Launch Cloud Shell, paste the commands below to simulate a device sending telemetry. You will be able to start monitoring events from the built-in event hub."
+            },
+            "prerequisite": {
+                "label": "Prerequisite",
+                "tooltiop": "If you have not used Azure Cloud Shell before",
+                "instruction": "Follow the instructions in Azure Cloud Shell to pick a subscription and set up a storage if you haven't used Cloud Shell before. If you have encountered error 'az iot: 'device' is not in the 'az iot' command group', use command 'az extension add --name azure-iot' to install Azure IoT CLI extension."
+            },
+            "basic": {
+                "label": "Basic scenario",
+                "tooltiop": "Zero input basic scenario",
+                "instruction": "Simply copy paste the following command to Azure Cloud Shell. It will start simulating device {{deviceId}} as it's sending messages to IoT Hub. You can click 'Start' button from the Telemetry page to start monitoring the events.",
+                "copyLabel": "Simulating device {{deviceId}}"
+            },
+            "advanced": {
+                "label": "Advance scenario",
+                "tooltiop": "For advanced users to simulate events' body or properties",
+                "instruction": "Enter the desired message body and/or properties, and copy paste the generated command to Azure Cloud Shell. It will start simulating device {{deviceId}} as it's sending messages to IoT Hub. You can click 'Start' button from the Telemetry page to start monitoring the events.",
```

Comment:
Quick take: If the user wants to provide system properties that the simulated device sends, should IoT explorer help them with that, or is the user expected to know the property syntax differences (which also change with protocol)?

Reply:
I am leaving it simply by asking users to input them, exactly due to your comment (they change with protocol). I'd like to see if user would use this feature to future inhance them, with enable cloud shell to create a hub for example.